### ISSUE TITLE
pre-trained model and instructions

### ISSUE THREAD

**Issue by AugustLee93** [liamheng/Restoration-of-Cataract-Images-via-Domain-Adaptation#1]
Hi, there, I was very impressed by your work ! Can you provide a pretrain model and give some instrucions of your work ? I would like to test how good the restoration and prediction will be.  Thank you very much!

**Comment by HeverLaw** [comment]
Thank you for your interest in our study. We have uploaded a trained model as a demo for the algorithm. README.md has been updated for instructions. Please connect us if there is any further question.

**Comment by AugustLee93** [comment]
> Thank you for your interest in our study. We have uploaded a trained model as a demo for the algorithm. README.md has been updated for instructions. Please connect us if there is any further question.

Thank you for your reply! I tried your pre-trained model, but I'm not sure what  your format of the dataset as input is. Isn't it supposed to be a single pic as input, or i need to format the inputs data like one pic but concatenate  a lable, like {cataract pic, clear cataract pic(lable)}?

**Comment by HeverLaw** [comment]
From your understanding, I think the former one it's right for the target dataset and the latter one it's right for the source dataset. A dataset contains two parts, including the source dataset and the target dataset. The source dataset contains the image pairs where a pair is constructed like {cataract image, clear fundus image(label)}, and the target dataset contains real cataract images. See the file structure in https://github.com/liamheng/Restoration-of-Cataract-Images-via-Domain-Adaptation/tree/master/images/cataract_dataset to get a more intuitive understanding.

**Comment by AugustLee93** [comment]
> From your understanding, I think the former one it's right for the target dataset and the latter one it's right for the source dataset. A dataset contains two parts, including the source dataset and the target dataset. The source dataset contains the image pairs where a pair is constructed like {cataract image, clear fundus image(label)}, and the target dataset contains real cataract images. See the file structure in https://github.com/liamheng/Restoration-of-Cataract-Images-via-Domain-Adaptation/tree/master/images/cataract_dataset to get a more intuitive understanding.

Thanks for your useful reply! But I still have a few question. Firstly, if in train stage, dataset contains source file like {cataract image, clear fundus image(label)}, I get it. But within a pre-trained model, if source file is empty ( but target file has a image to test), then I got None result output.  I was wondering how it works. By the way, I tried if source and target file contain images, then the  test image in target file has a good result, good work!
